### DISCUSSION TITLE
How to interpret fecundity in the transition matrix

### DISCUSSION THREAD

**Discussion by anyperez** [RangeShifter/RangeshiftR-tutorials#8] [Q&A] [answered]
Hello again,

Quoting the information on fecundity in the manual, "In a female-only model (ReproductionType=0), φ represents the **number of female offspring** per female and reproductive event.", and as in the examples of Box1-4 in the manual that the value of φ is 1.5, 3, 5, etc, clearly the fecundity is not taken as an actual probability (although the software uses a Poisson distribution to account for stochasticity). 

If I want to consider the fecundity in the transition matrix as the probability that a female in a certain stage reproduces, what should I do? Is it possible?

Thank you

**Comment by damariszurell** [comment, marked as answer]
Hi Any,

As you should see in the different tutorials provided in the manual and online under https://rangeshifter.github.io/RangeshiftR-tutorials/, fecundity in matrix model annotation describes the average number of offspring. So, yes, it is not a probability. If you have separate information on probability to carry offspring and on number of offspring, then you have to calculate yourself how the final matrix would need to look like.

**Comment by annekathleenmalchow** [comment]
Hi,
just to add: Like you have read in the manual, the fecundity φ is used as the expected value in a Poisson distribution, from which the realised number of offspring is drawn in an reproductive event. It is set in the first line of the transition matrix. However, if you want to set the probability with which a female starts a reproductive event in the first place, you can use the parameter 'PRep' that sets the probability of reproducing (per reproductive season).
